tensorboard/plugins/pr_curve/pr_curve_plugin_test.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests the op that generates pr_curve summaries."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import json
+import os
+
+import numpy as np
+import tensorflow as tf
+
+from tensorboard.backend.event_processing import event_multiplexer
+from tensorboard.plugins.pr_curve import pr_curve_op
+from tensorboard.plugins.pr_curve import pr_curve_plugin_data_pb2
+
+
+class PrCurveTest(tf.test.TestCase):
+
+  def setUp(self):
+    self.logdir = self.get_temp_dir()
+    tf.reset_default_graph()
+
+  def test1Class(self):
```

Comment:
I added a test for precision and recall across the thresholds. My current plan is to have the frontend (JS) compute the precision and recall - the backend could send the counts to the frontend. That way, the user could say hover over the PR curve and see specific TP, FP, TN, FN values.